core/templates/dev/head/css/oppia.css

```diff
@@ -4709,3 +4709,7 @@ md-card.preview-conversation-skin-supplemental-card {
   padding: 0;
   width: 1px;
 }
+
+.collection-editor-exp-search {
+  width: 210px !important;
```

Comment:
Is is possible to change it so that we don't have `!important` (ie. move this to the html so that's an inline attribute)?